tracee-ebpf/Makefile

```diff
@@ -124,6 +124,10 @@ test: $(DOCKER_BUILDER)
 	$(call docker_builder_make,$@)
 endif
 
+.PHONY: test-integration
+test-integration:
+	$(go_env) TRC_BIN=../$(OUT_BIN) go test -exec 'sudo -E' -v integration/*_test.go
```

Comment:
why is the `../` prefix required? I think it should be `TRC_BIN=$(OUT_BIN)` no?

Reply:
I suspect this has to do with running from the root makefile, in this case probably that makefile should set TRB_BIN to the correct location

Reply:
Actually I believe this path is relative to the integration module not to the Makefile. That's why going up a directory is needed. Go tests run relative to the directory they are in. 

> in this case probably that makefile should set TRB_BIN to the correct location

Isn't that what the Makefile is doing now? Or did you have something else in mind.